mod/layout/set/abstract/thumbnail.rb

```diff
@@ -9,12 +9,12 @@
     end
   end
 
-  view :thumbnail do
+  view :thumbnail do |args|
```

Comment:
so how can I pass arguments to another view with voo? I don't see that implemented?